docs/migrations/v0.32.md

```diff
@@ -0,0 +1,62 @@
+# v0.30
+
+[Release notes](https://github.com/grafana/grafana-app-sdk/releases/tag/v0.32.0)
+
+*Upgrading from*: v0.30.0, v0.31.*
+
+## What Changed from v0.30.0 and v0.31.*
+
+### CLI Changes (non-breaking)
+
+The flag `--noschemasinmanifest` has been introduced to not generate OpenAPI schemas in the generated go manifest. 
```

Comment:
nit: this may be clearer?
```suggestion
The flag `--noschemasinmanifest` has been introduced, which can be used to prevent OpenAPI schemas from being generated in the go manifest. 
```